models/topics.models.js

```diff
@@ -0,0 +1,8 @@
+const db = require('../db/connection.js')
+exports.selectTopics= () => {
+    let queryString = `SELECT * FROM topics`
```

Comment:
Do you ever reassign this, is let the best choice?

Reply:
changed to const queryString on line3